### ISSUE TITLE
Question: roadmap

### ISSUE THREAD

**Issue by raffaeler** [danielgerlag/workflow-core#1157]
I found this project and it looks really great.
Anyway, there are a lot of PRs and I see no releases since 2021.
What is the roadmap for this project? Is it still actively developed?

Thank you

**Comment by wfcore-manager** [comment]
Yes — still actively developed. Latest release is 3.19.0: https://www.nuget.org/packages/WorkflowCore/3.19.0
